src/NHSD.BuyingCatalogue.Identity.Api/Views/Account/ForgotPassword.cshtml

```diff
@@ -0,0 +1,25 @@
+﻿@model NHSD.BuyingCatalogue.Identity.Api.ViewModels.Account.ForgotPasswordViewModel
+
+<div class="nhsuk-width-container">
+    <main class="nhsuk-main-wrapper">
+        <div data-test-id="login-page">
+            <div class="nhsuk-grid-row">
+                <div class="nhsuk-grid-column-two-thirds">
+                    <h1 data-test-id="page-title" class="nhsuk-heading-xl nhsuk-u-margin-bottom-2">Enter your email address</h1>
```

Comment:
Please have a look at the Login.cshtml page on how it sets the page title and page description because in the _Layout file it uses those values within the ViewData to set various page properties.

Reply:
Do we have content for these? I can't find it